connectors/gitlab-pr/src/test/resources/fake/cards/big-closed-pr.json

```diff
@@ -9,7 +9,7 @@
   "author": {
     "username": "TestUser1"
   },
-  "sha": "9fbf614c4c5c76ac93e82c59976a8be5f0c0117a",
+  "sha": "test-sha",
```

Comment:
Unfortunately, this is what happens when we have all of our tests relying on json files and string replace...